packages/core-sdk/src/utils/oov3.ts

```diff
@@ -23,3 +23,18 @@ export const getAssertionDetails = async (
 
   return bond;
 };
+
+export const getMinimumBond = async (
```

Comment:
How are you handling the error if the contract call fails? perhaps use 'try/catch'?

Reply:
this should not be needed, the caller should be handling errors. I think its currently handled well.